codersdk/deployment.go

```diff
@@ -2028,8 +2029,26 @@ func (e Experiments) Enabled(ex Experiment) bool {
 	return false
 }
 
-func (c *Client) Experiments(ctx context.Context) (Experiments, error) {
-	res, err := c.Request(ctx, http.MethodGet, "/api/v2/experiments", nil)
+type ExperimentOptions struct {
+	// All signifies that all experiments - rather than just those that are enabled -
+	// should be returned
+	IncludeAll bool `json:"include_all,omitempty"`
```

Comment:
This comment is incorrect, since it omits the enabled experiments. It only returns the "safe" experiments, but more can be set.